src/ui/components/SecondaryToolbox/redux-devtools/ReduxDevToolsPanel.tsx

```diff
@@ -60,6 +61,12 @@ export default function ReduxDevToolsPanel() {
     return [filteredAnnotations, firstAnnotationAfterCurrentExecutionPoint];
   }, [annotationsStatus, currentExecutionPoint, focusWindow, parsedAnnotations, searchValue]);
 
+  useEffect(() => {
+    // Preload the cache for where `applyMiddleware` is defined
+    // This will speed up the first click on a Redux "J2C" button
+    applyMiddlewareDeclCache.readAsync(client);
```

Comment:
nit: `suspense` offers `Cache.prefetch()` for this scenario: the only difference is that `Cache.prefetch()` will catch errors (and doesn't return anything).

Reply:
Ah, yes, that _does_ exist!  Okay, will throw that in.